### DISCUSSION TITLE
show markdown file name in slide when in draft mode

### DISCUSSION THREAD

**Discussion by emicklei** [marp-team/marp#179] [Ideas]
Hi,

While working on a large slide deck, I realised I spent a bit of time searching for the source file of a slide to make changes. 
How about showing the name (or link) to the markdown file that was used per slide? Obviously this should only be shown when creating the deck, not while presenting it. Hence the "draft" mode.

**Comment by yhatt** [comment]
I have thought that this idea is good to start from a Marp plugin. 

Marp is not providing a context about a source file of Markdown because Marp is not always used conversion with a Markdown resource that have a specific path. (e.g. Use [Marp Core](https://npm.im/@marp-team/marp-core) programmatically)

But if Marp CLI provided the path to source, community can make a plugin to add a watermark with a link to source file when the parsed Markdown has `<!-- draft: true -->` global directive.

If someone of Marp plugin developers have interested in this idea, we can consider to add the context of source file to Marp CLI.
